packages/xrpl/src/Wallet/fundWallet.ts

```diff
@@ -20,15 +20,103 @@ const INTERVAL_SECONDS = 1
 const MAX_ATTEMPTS = 20
 
 /**
- * Generates a random wallet with some amount of XRP (usually 1000 XRP).
+ * The fundWallet() method is used to send an amount of XRP (usually 1000) to a new (randomly generated)
+ * or existing XRP Ledger wallet.
  *
  * @example
- * ```typescript
+ *
+ * Example 1: Fund a manually random generated wallet
+ * const { Client, Wallet } = require('xrpl')
+ *
+ * const client = new Client('wss://s.altnet.rippletest.net:51233')
+ *
+ * async function fundRandomWallet() {
+ *   const newWallet = new Wallet()
```

Comment:
Change to `Wallet.generate()`

Reply:
are those comments the changes you plan to make? or you are asking us if you should make it

Reply:
Making changes